dev/archery/archery/integration/runner.py

```diff
@@ -627,6 +627,11 @@ def run_all_tests(with_cpp=True, with_java=True, with_js=True,
             description="Ensure Flight SQL extensions work as expected.",
             skip_testers={"Rust"}
         ),
+        Scenario(
+            "flight_sql:ingestion",
+            description="Ensure Flight SQL ingestion works as expected.",
+            skip_testers={"JS", "C#", "Rust", "Java"}
+        ),
```

Comment:
Are scenarios documented somewhere or is there a plan to do it? @lidavidm

Reply:
No, what would you like to see?

Reply:
Either a subsection in https://arrow.apache.org/docs/dev/format/Integration.html or a separate page dedicated to Flight integration testing?

Reply:
https://github.com/apache/arrow/issues/38646